pkg/keycloak/client.go

```diff
@@ -415,6 +534,11 @@ type KeycloakInterface interface {
 	DeleteClient(clientId, realmName string) error
 	UpdateClient(kcClient, objClient *v1alpha1.KeycloakClient, realmName string) error
 	ListClients(realmName string) ([]*v1alpha1.KeycloakClient, error)
+
+	CreateUser(user *v1alpha1.KeycloakUser, realmName string) error
+	DeleteUser(userID, realmName string) error
+	UpdateUser(kcUser, objUser *v1alpha1.KeycloakUser, realmName string) error
```

Comment:
Why does update need both kcUser and objUser here? Is this due to the issue with not having the generated ID on the spec object?

Reply:
Yes, but I'll move that to `keycloak.go`

Reply:
Although, I don't really mind the way it is now